microsoft/atlas-design
# init js sharing infrastructure

Task: task-[work-item-number]

Link: preview-374

The publishing mechanism can only _really_ be verified after this PR has merged. However, we can also double check that change haven't affected anything negatively in the site.

## Testing

1. Visit site and popover page. 
2. Verify JS still working fine.
3. Run the following command and inspect generated files in js/dist

```
npm run build:js
```
